asserts/account_key_test.go

```diff
@@ -218,8 +240,51 @@ func (aks *accountKeySuite) TestAccountKeyCheck(c *C) {
 
 	db := aks.openDB(c)
 
+	aks.prereqAccount(c, db)
+
+	err = db.Check(accKey)
+	c.Assert(err, IsNil)
+}
+
+func (aks *accountKeySuite) TestAccountKeyCheckNoAccount(c *C) {
```

Comment:
I think this test is an example of where a NewAccountKey that didn't take a *Account as a param would help remove noise from the test.